apps/to-do-api/src/adapters/azure/functions/get-tasks.ts

```diff
@@ -0,0 +1,30 @@
+import * as H from "@pagopa/handler-kit";
+import { httpAzureFunction } from "@pagopa/handler-kit-azure-func";
+import * as RTE from "fp-ts/lib/ReaderTaskEither.js";
+import * as RA from "fp-ts/lib/ReadonlyArray.js";
+import { flow, pipe } from "fp-ts/lib/function.js";
+
+import { Capabilities } from "../../../domain/Capabilities.js";
+import { listTasks } from "../../../domain/TaskRepository.js";
+import { TaskItemList } from "../../../generated/definitions/internal/TaskItemList.js";
+import { toHttpProblemJson, toTaskItemAPI } from "../../http/codec.js";
+
+type Env = Pick<Capabilities, "taskRepository">;
+
+const makeHandlerKitHandler: H.Handler<
+  H.HttpRequest,
+  | H.HttpResponse<H.ProblemJson, H.HttpErrorStatusCode>
+  | H.HttpResponse<TaskItemList>,
+  Env
+> = H.of(() =>
+  pipe(
+    RTE.ask<Env>(),
+    // execute use case
+    RTE.flatMap(listTasks),
```

Comment:
Note: in this case, the `listTasks` isn't a use case but a function under the `domain` layer.

this is wanted because the use case would be just a wrapper around the function already used, the `listTasks`.
If the `list` operation is going to be more complicated, with some custom logic before sending the result, then a use case would be the place where to hide that logic.